src/test/java/org/cactoos/http/HtTimedWireTest.java

```diff
@@ -39,11 +39,12 @@
  * @checkstyle JavadocMethodCheck (500 lines)
  * @checkstyle JavadocVariableCheck (500 lines)
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ * @checkstyle MagicNumberCheck (500 lines)
```

Comment:
@iakunin please keep the exception like this close to the place where they happen

Reply:
@victornoel ok, that's not a problem